tasks/main.yml

```diff
@@ -1 +1,60 @@
 # role tasks
+
+- name: Check localectl status before
+  command: localectl status
+  register: localectl
+  changed_when: "localectl.rc != 0"
```

Comment:
If the rc is non-zero, ansible will fail this task and the entire playbook that contains it.

This is an inappropriate use of `changed_when`. `changed_when` should be used to tell ansible when it's appropriate to report that a task made a change. As this is a "check" task, it would ideally never report a change.

Reply:
All instances of `changed_when` could likely be replaced by a handler.  Basically the flow would be:

- Check status first, compare to values defined by the user
- If the current values don't match what the user wants, run a handler to make the change, otherwise do nothing, and report unchanged status.

Reply:
This particular case, since `localectl status` doesn't ever change anything the proper line is `changed_when: false`.